src/styles/blockCard.scss

```diff
@@ -0,0 +1,136 @@
+.container {
+  display: grid;
+  max-width: 1200px;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  margin: 0;
+  gap: 46px 48px;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.block {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+
+  transition:
+    transform 0.3s ease,
+    box-shadow 0.3s ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 160px 134px;
```

Comment:
Can be simplified by using one property `background`
https://developer.mozilla.org/en-US/docs/Web/CSS/background